### ISSUE TITLE
NOT WORKING ANYMORE

### ISSUE THREAD

**Issue by onexoff** [mowshon/remove-disqus-ads#1]
HELLO COULD YOU FIX IT PLEASE I HAVE TESTED IT ON THE LAST WORDPRESS VERSION AS DISQUS TOO IS NOT WORKING ANYMORE

**Comment by mowshon** [comment]
@onexoff did you disable ads from admin panel from disqus?
Also, if you have something like https://wordpress.org/plugins/disqus-conditional-load/
You'll need to disable such "Conditional Load" plugin.

**Comment by onexoff** [comment]
yes currently I'm using Conditional Load  I've included the class of
remove-disqus-ads inside
 Conditional load it should work as I think
  do you have any idea to how to fix it

**Comment by mowshon** [comment]
@onexoff yeah, try to disable Conditional Load

**Comment by vsatyamesc** [comment]
Disbaled it but still not working
